javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,82 @@
+/////// Movies exercise
+const moviesUrl = 'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json'
+
+fetch(moviesUrl)
+    .then(response => response.json())
+    .then(movies => {
+        console.log(movies);
+        
+    // bad movies array
+        const badMovies = movies.filter(movie => movie.rating < 4);
+        console.log('bad movies', badMovies);
+    
+        const badMoviesSince2000 = movies.filter(movie => movie.year >= 2000);
+        console.log('bad movies since 2000', badMoviesSince2000);
```

Comment:
It shows all movies (with all rates) since 2000, so If you want to have just bad movies since 2000, then you can make a filter on bad movies. :)